frontend/package.json

```diff
@@ -22,6 +22,7 @@
     "@radix-ui/react-slot": "^1.0.2",
     "@radix-ui/react-tooltip": "^1.1.0",
     "@radix-ui/themes": "^3.0.3",
+    "@sindresorhus/slugify": "^2.2.1",
```

Comment:
You can remove this package as it is not used yet, don't forget to readd the `pnpm-lock` file to git as it will modify it again.